.github/workflows/ci.yml

```diff
@@ -33,7 +33,9 @@ env:
   INTEG_TEST_MAPPING_DOWNSTREAM_FAILURE_TOPIC: 'mapping-failure-bigquery'
   INTEG_TEST_POLICY_UPSTREAM_TOPIC: 'policy-gcs'
   INTEG_TEST_POLICY_DOWNSTREAM_TOPIC: 'policy-bigquery'
-  INTEG_TEST_STATIC_GCS_BUCKET: 'pmap-static-ci-bucket-7faa'
+  INTEG_TEST_STATIC_GCS_BUCKET_CI: 'pmap-static-ci-bucket-7faa'
+  INTEG_TEST_STATIC_GCS_BUCKET_DEV: 'pmap-dev'
```

Comment:
Maybe create a dedicated gcs bucket for this e2e test and use specific naming.

Reply:
Also, we can create it in ci project.